Documentation/how-to-build-and-run-ilcompiler-in-console-shell-prompt.md

```diff
@@ -11,7 +11,7 @@ build[.cmd|.sh] clean [Debug|Release]
 This will result in the following:
 
 - Restore nuget packages required for building
-- Build native and managed components of ILCompiler. The final binaries are placed to `<repo_root>\bin\Product\<OS>.<arch>.<Config>\.nuget\publish1`.
+- Build native and managed components of ILCompiler. The final binaries are placed to `<repo_root>\bin\Product\<OS>.<arch>.<Config>\.nuget\publish1\Debug\dnxcore50`.
```

Comment:
Isnt the "Debug" dependent upon whether CLI was specified Release or Debug? Shouldn't it be "<BuildType>" instead?